GetFileInfo: add page (#4004)
tldr-pages/tldr: pages/osx/GetFileInfo.md

# GetFileInfo

> Get information about a file in an HFS+ directory.

> - See date and time of file creation:

Reviewer: This sounds really odd. Perhaps this will read better?
```suggestion
- Display the date and time a given file was created:
```

> `GetFileInfo -d {{filename}}`

Reviewer: Perhaps this will read better? Then it's more consistent with existing pages :-)
```suggestion
`GetFileInfo -d {{path/to/filename}}`
```

Also, is there a long form of the `-d` argument here? In tldr pages we prefer to use the long forms of arguments to aid clarity. The same goes for the other examples further down.

Author: @sbrl, I don't think there is a long-form for any of them. The man page of GetFileInfo does not list any (and I tried to expand them out myself too). Do let me know if you have any suggestions on how to provide more clarity.

Reviewer: No problem - just checking, since some commands do have long forms and others don't :-)

> - See date and time of latest file modification:

Reviewer: Perhaps this will read better?
```suggestion
- Display the date and time a given file was last modified:
```

> `GetFileInfo -m {{filename}}`

Reviewer: Consistency again (same goes for the other two examples below).
```suggestion
`GetFileInfo -m {{path/to/filename}}`
```

> - See the creator of the file:

Reviewer: Perhaps this will read better?
```suggestion
- Display the creator of the given file:
```

`GetFileInfo -c {{filename}}`

> - See all infomation about the file:

Reviewer: Perhaps this will read better?
```suggestion
- Display information about the file:
```

Also, perhaps this should be the first example, not the last?
